docs/preview_conversion_analyzer.md

```diff
@@ -0,0 +1,129 @@
+<!-- The purpose of this spec is to describe a new feature and
+its APIs that make up a new feature in WinUI. -->
+
+<!-- There are two audiences for the spec. The first are people
+that want to evaluate and give feedback on the API, as part of
+the submission process.  When it's complete
+it will be incorporated into the public documentation at
+docs.microsoft.com (http://docs.microsoft.com/uwp/toolkits/winui/).
+Hopefully we'll be able to copy it mostly verbatim.
+So the second audience is everyone that reads there to learn how
+and why to use this API. -->
+# Converting To WinUI3 Using CSharp Conversion Analyzers
+
+## Background
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+The purpose of this project is to create a porting solution in the form of Code Analyzers/Fixes to allow Developers to convert WinUI2 projects to the new WinUI3 format. As Microsoft moves toward a purely public SDK, its Xaml UI components are being lifted out of the OS and deployed in WinUI3.
```

Comment:
Could you say more about the 'what' before the 'how' (code analyzers)? And then explain what a code analyzer/fix is (that it's an extension to the Visual Studio experience).

Reply:
Updated background to be more thorough.